pkg/kubectl/cmd/expose.go

```diff
@@ -162,6 +162,10 @@ func RunExpose(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []str
 		}
 		params["labels"] = kubectl.MakeLabels(labels)
 	}
+	if generatorName == "service/v1" {
```

Comment:
Yeah, this is exactly why @bgrant0607 made me create the generator stuff in the first place...